docs/integrations/sources/postgres.md

```diff
@@ -400,6 +400,7 @@ The root causes is that the WALs needed for the incremental sync has been remove
 
 | Version | Date       | Pull Request                                                                                                       | Subject                                                                                                                                                                    |
 |:--------|:-----------|:-------------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 1.0.24  | 2022-11-07 | [19291](https://github.com/airbytehq/airbyte/pull/19291)                                                           | Use native Postgres connection timeout                                                                                                                                                       |
```

Comment:
Can you just add a detail saying default timeout is reduced from 1 min to 10sec

Reply:
> Can you just add a detail saying default timeout is reduced from 1 min to 10sec

done